docs/file_format.rst

```diff
@@ -38,6 +38,13 @@ The optional ``extracellular`` key specifies the default string for
 the extracellular compartment on compounds. If this option is not
 specified it will be assumed that the extracellular compartment is called ``e``.
 
+-Default Compartment
+--------------------------
+
+The optional ``default`` key specifies the default string for
+the default compartment on compounds. If this option is not
```

Comment:
Maybe we can change this a bit to clarify what it does: "The optional ``default`` key specifies the default compartment that is used if a compound in a reaction does not explicitly specify a compartment." Also, shouldn't this be `default_compartment`? I think `default` is too ambiguous as a key.